internal/driver/macmapper.go

```diff
@@ -7,16 +7,17 @@
 package driver
```

Comment:
There are a few comments missed in this file
```
//UpdateMappings takes the raw map of secret path to csv list of mac addresses and
// inverts it into a quick lookup map of mac address to secret path.
// the default secret path if the mapping is not found, or no auth if the mac address is invalid.
// if the secret path is the special NoAuth magic key, do not look it up, instead return the noAuthCredentials
// the default secret path will be used to look up the credentials. An error is returned if the secret path
```

Reply:
or @lenny-intel can you help to complete the comments?

Reply:
@vli11 , that is just an artifact of how @ChristianDarr-personal is showing the missed updates. 
All you need to do is search for `path` in the file and update the missed references to `path`

Reply:
My bad, some of the comments got chopped off - I was just hoping to show the comments that had `path` in them